testpilot/frontend/static-src/app/views/retire-page.js

```diff
@@ -56,6 +56,12 @@ export default PageView.extend({
         'X-CSRFTOKEN': app.me.csrfToken
       },
       body: ''
+    }).then(() => {
+      app.sendToGA('event', {
```

Comment:
I'm addressing this in [my PR](https://github.com/mozilla/testpilot/pull/681/commits/f923cf019b32f10e03e6a80d484de238c4cef279#diff-91e7152e270794146b7cb685abcb5667R59) as well. Race?